### PR TITLE
feat: bump TRIVYVERSION to v0.32.1 and bump TRIVYADAPTERVERSION to v0.30.2

### LINKED ISSUE #17525: No method to disable secrets vulnerability scanning, leading to slow scans on large images
**Expected behavior and actual behavior:**

Recent Trivy version introduced secrets vulnerability scanning that is enabled by default and slows down scanning especially for large images.

However there is no way to set security-checks since the adapter doesn't handle it:

https://github.com/aquasecurity/harbor-scanner-trivy/blob/ff073cd57730f9258ec9fb6a72852f61a598f6e4/pkg/etc/config.go#L26-L39

I believe trivy-adapter should set `--security-checks vuln` by default since `secrets` vulns are not reported back to Harbor anyway, or at least allow the user to set `SCANNER_TRIVY_SECURITY_CHECKS` env variable to adjust it.

adapter logs also have this (for failed scans):
```
If your scanning is slow, please try '--security-checks vuln' to disable secret scanning\n2022-09-08T13:09:17.626Z\t\u001b[34mINFO\u001b[0m\tPlease see also https://aquasecurity.github.io/trivy/v0.29.2/docs/secret/scanning/#recommendation for faster secret detection
```

**Steps to reproduce the problem:**

Scan a large image (>3~4GB) and observe the issue

**Versions:**
Please specify the versions of following systems.

- harbor version: v2.5.3
- docker engine version: N/A
- docker-compose version: N/A

**Additional context:**

/cc @danielpacak

### PR DESCRIPTION
**Pull request by zyyw**
Signed-off-by: Shengwen Yu <yshengwen@vmware.com>

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #17525

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by stonezdj — APPROVED** [review]

File: make/harbor.yml.tmpl [hunk 1]
```diff
@@ -86,6 +86,9 @@ trivy:
   # This option doesn’t affect DB download. You need to specify "skip-update" as well as "offline-scan" in an air-gapped environment.
   offline_scan: false
   #
+  # Comma-separated list of what security issues to detect. Possible values are `vuln`, `config` and `secret`. Defaults to `vuln`.
+  security_check: vuln
```
Comment (chlins) [inline make/harbor.yml.tmpl:90]: If I want to enable `config` or `secret`, what should input? `vuln, secret`?

Comment (zyyw) [reply make/harbor.yml.tmpl:90]: correct. comma-separated without space:

Comment (chlins) [reply make/harbor.yml.tmpl:90]: Thanks, maybe we can add comments for this.

**Review by chlins — APPROVED** [review]
lgtm

**Pull request merged**